simple/src/main/java/org/apache/commons/rdf/simple/experimental/AbstractRDFParser.java

```diff
@@ -240,7 +240,7 @@ public T rdfTermFactory(RDF rdfTermFactory) {
     public T contentType(RDFSyntax rdfSyntax) throws IllegalArgumentException {
         AbstractRDFParser<T> c = clone();
         c.contentTypeSyntax = Optional.ofNullable(rdfSyntax);
-        c.contentType = c.contentTypeSyntax.map(syntax -> syntax.mediaType);
+        c.contentType = c.contentTypeSyntax.map(syntax -> syntax.mediaType());
         return c.asT();
```

Comment:
It hurts my head thinking about this, but what is the signature for the generic type "T" designed to do. It seems like a circular definition, which doesn't really make sense but I can't think why T could have been introduced so I am stuck at seeing it as circular.

```
AbstractRDFParser<T extends AbstractRDFParser<T>>
```